ui/components/component-library/avatar-icon/avatar-icon.js

```diff
@@ -4,7 +4,8 @@ import classnames from 'classnames';
 import {
   BorderColor,
   Size,
-  DISPLAY,
+  // DISPLAY,
```

Comment:
Looks like this comment needs to be removed.

Reply:
👍
Done Sir.